cirq/value/angle.py

```diff
@@ -88,7 +88,7 @@ def canonicalize_half_turns(
         half_turns: Union[Symbol, float]
 ) -> Union[Symbol, float]:
     """Wraps the input into the range (-1, +1]."""
-    if isinstance(half_turns, Symbol):
+    if isinstance(half_turns, sympy.Basic):
```

Comment:
There might be lots of different `sympy` things, right? We should invert this logic so that it cares about "not int and not float" instead of "is a specific brand of weird".